### PR TITLE
Set VS Code to user workspace TypeScript.

### PR DESCRIPTION
**Pull request by tcobbs-bentley**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "files.insertFinalNewline": true
-}
\ No newline at end of file
+  "files.insertFinalNewline": true,
+  "typescript.tsdk": "node_modules/typescript/lib"
```
Comment (toddsouthenbentley) [inline .vscode/settings.json:3]: Why is this needed? According to the [docs](https://code.visualstudio.com/docs/typescript/typescript-compiling): VS Code will automatically detect workspace versions of TypeScript that are installed under node_modules in the root of your workspace

Comment (tcobbs-bentley) [reply .vscode/settings.json:3]: It may detect the TypeScript version, but it doesn't use it for its autosense. And that leads to it underlining stuff like every single <div ...> in a tsx file, because it's using the wrong version of TypeScript for it code parsing.

**Review by toddsouthenbentley — APPROVED** [review]
